frontend/packages/dev-console/src/components/import/devfile/DevfileStrategySection.tsx

```diff
@@ -47,11 +50,25 @@ const DevfileStrategySection: React.FC = () => {
       setFieldValue('devfile.devfileHasError', true);
       setValidated(ValidatedOptions.error);
     } else {
+      fireTelemetryEvent('Download Devfile from Git', {
+        client: ODC_TELEMETRY_CLIENT_NAME,
```

Comment:
You may was well inline the constant since it's only use once.

Also wouldn't you want to fire the event only when the form is submitted?

Reply:
I was trying to future proof in case we wanted to introduce additional events in the future.   I made the changes anyway.  We're trying to capture download actions even if the client does not wind up deploying the sample.